docs/compatibility.md

```diff
@@ -489,6 +489,14 @@ these formats when unquoted, will produce `null` on the CPU and may produce vali
 Another limitation of the GPU JSON reader is that it will parse strings containing floating-point values where
 Spark will treat them as invalid inputs and will just return `null`.
 
+### json options
+
+Below are some of the options that are supported by the plugin with minor differences when compared to Spark:
```

Comment:
Can we explain this a little more. Perhaps something like.

```
Spark supports passing options to the JSON parser when reading a dataset.  In most cases if the RAPIDS Accelerator sees one of these options that it does not support it will fall back to the CPU. In some cases we do not. The following options are documented below.
```